library/src/main/assets/leaflet_map.html

```diff
@@ -486,13 +494,17 @@
 }
 
 function getCenter() {
-  var latLng = latLngFromPaddingLatLng(map.getCenter());
+  var latLng = getPaddingCenter();
   if (latLng != null) {
     var convertedLatLng = reverseConvertLatLng([ latLng.lat, latLng.lng ]);
     AirMapView.mapCenterCallback(convertedLatLng[0], convertedLatLng[1]);
   }
 }
 
+function getPaddingCenter() {
```

Comment:
Like I said above, I guess this is center of the view with padding?

Reply:
Done